### DISCUSSION TITLE
How to setup a oos compatible storage on Harbor(Verison 2.6.1)?

### DISCUSSION THREAD

**Discussion by fzwu** [goharbor/harbor#17760] [Q&A]
Hi.

I'm trying to use an OOS compatible S3 bucket as my storage backend for harbor, and I've set this in harbor.yml file.

storage_service:
  oos:
    accesskeyid: xxxxxxx
    secretkeysecret: xxxxxxx
    bucket: 20221104
    endpoint: https://obs.cn-east-3.myhuaweicloud.com
    region: obs.cn-east-3
    rootdirectory: /
 redirect:
    disabled: true

I have pushed the mirror image to harbor,But the oos has no files,What should I do for config the harbor.yml?

**Comment by fzwu** [comment]
Has been resolved.

**Reply by Vad1mo** [reply]
@fzwu can you share your results and config what worked for you?

**Reply by fzwu** [reply]
I have solved the problem

**Reply by Vad1mo** [reply]
@fzwu 
We got that. Sharing is caring. So sharing your results and findings back with the community, we can ensure that the knowledge becomes a public good and other people can build upon it or improve it, take care of it in the form of documentation. 

This is the engine that keeps the open source projects working.

**Comment by MinerYang** [comment]
Any specific tips for community users for using oos s3 compatible storage? Appreciate!

**Reply by fzwu** [reply]
I have solved the problem
